zebrad/tests/acceptance.rs

```diff
@@ -1455,7 +1455,73 @@ async fn tracing_endpoint() -> Result<()> {
     Ok(())
 }
 
-// TODO: RPC endpoint and port conflict tests (#3165)
+#[tokio::test]
+async fn rpc_endpoint() -> Result<()> {
```

Comment:
I think this test name will be more descriptive as `lightwalletd_endpoint` if we do https://github.com/ZcashFoundation/zebra/pull/3624#discussion_r813443329

Reply:
Go for it!

Reply:
Yea, maybe i will just leave it because if i do the rename i will be tempted to change more, like `zebra_rpc_conflict` test name, strings as `Opened RPC endpoint at` to `Opened lightwalletd endpoint at`, etc. I dont think this is what we want.

Lets keep it as RPC and use valid lightwalletd requests.